various-receipes/typescript/test/typescript-example.ts

```diff
@@ -0,0 +1,21 @@
+import * as express from "express";
+
+import { initializeWebServer, stopWebServer } from "../../../example-application/api-under-test";
+
+let expressApp: express.Application;
+
+beforeAll(async () => {
+  expressApp = await initializeWebServer();
+});
+
+afterAll(async () => {
+  await stopWebServer();
+});
+
+describe("typescript", () => {
+
+    test("should be able to compile and run", () => {
+      expect(true).toBeTruthy();
```

Comment:
@DanielGluskin What are we showing here? Just an empty file? Mostly the config?

Reply:
Yep, ts-jest and such. What was your original intention? Do you plan to include typescript inside the best practices?

I think we shouldn't insert any logic here bacause it will result in code-duplication and there is no difference between the test code itself.

Reply:
Maybe it's minor but TypeScript tests have types, the code is slightly different and we could theoretically duplicate the basic.test or one test. Probably not too important.